conseil-api/src/main/resources/application.conf

```diff
@@ -28,7 +40,10 @@ conseil {
   security.apiKeys {
     # List of authorized API keys needed to query Conseil
     keys: []
+    keys+= ${?CONSEIL_API_SECURITY_KEY}
```

Comment:
Please refer to this as CONSEIL_API_KEYS

Reply:
I believe, you can only specify a "KEY" not "KEY**S**". At least I don't know how to do that (meaning pass a list via this macro). If you know that it's a matter of changing `keys+= ${?CONSEIL_API_KEYS}` into `keys: ${?CONSEIL_API_KEYS}` then I can change that. By the way, I've added `SECURITY` in a path, just to reflect new section in it.

Reply:
oh, I didn't realise that... is that valid HOCON? I mean, the `+=` operation?

Ok, for the security section, yet when we talk about api keys from the user perspective it was never referred to as security keys... that would create some confusion?

Reply:
Yes it is valid option. I was surprised too. 
Definition: `+= appends an element to a previous array. If the previous value was not an array, an error will result just as it would in the long form a = ${?a} [b]. Note that the previous value is optional (${?a} not ${a}), which allows a += b to be the first mention of a in the file (it is not necessary to have a = [] first).`

About second one, alright, without `SECURITY` then :)